hack/e2e.sh

```diff
@@ -2,7 +2,19 @@
 
 set -euo pipefail
 
-kubectl kuttl test
+# Path to a clouds.yaml to use for e2e tests.
+# Exported because it is referenced in kuttl tests.
+export E2E_OSCLOUDS=${E2E_OSCLOUDS:-/etc/openstack/clouds.yaml}
+
+# Run kuttl tests from a specific directory.
+# Defaults to empty string (all discovered kuttl directories)
+E2E_KUTTL_DIR=${E2E_KUTTL_DIR:-}
+
+# Run a specific kuttl test.
+# Defaults to empty string (run all tests)
+E2E_KUTTL_TEST=${E2E_KUTTL_TEST:-}
+
+kubectl kuttl test "$E2E_KUTTL_DIR" --test "$E2E_KUTTL_TEST"
```

Comment:
have you checked that it runs the full test suite when you don't set either `E2E_KUTTL_DIR` or `E2E_KUTTL_TEST`?

Reply:
Gave it a try and:
- `kubectl kuttl test --test ""` works as expected and runs all tests
- however, `kubectl kuttl test ""` won't find any of the tests as it's only looking in the current directory and ignores the setting from `kuttl-test.yaml`. We need to either drop the quotes, or have a conditional for when the variable is set.